### ISSUE TITLE
Data Table sortable needs to be set to True

### ISSUE THREAD

**Issue by mrpanda** [evidence-dev/evidence#1784]
## Request Type

- [X] Small Fix (wording, errors, typos, etc.)

## Description

Documentation states that Data Table "sortable" option default = True. However, sorting does not work unless sortable=true in Data Table.

Either doco error (default = False ?) or fix code.

**Comment by archiewood** [comment]
Hi @mrpanda - I can't reproduce this

![CleanShot 2024-03-25 at 18 16 22](https://github.com/evidence-dev/evidence/assets/58074498/31b6a348-b329-482d-b22d-d5118955d66d)

What platform is this on?

**Comment by mrpanda** [comment]
Hi Archie, I have found the problem. The build process was still running in the background. If I had waited another 5 seconds then I would have seen this. Sorry, the computer must have been slow at that time and I jumped to conclusions, because I added 'sortable=true' and it started working straight away. I have checked several times now before writing this, issue is fine and I am on latest version of Evidence.
